src/common/window-utils.ts

```diff
@@ -25,7 +25,7 @@ export class WindowUtils {
     }
 
     public setTimeout(handler: Function, timeout: number): number {
-        return window.setTimeout(handler, timeout);
+        return globalThis.setTimeout(handler, timeout);
```

Comment:
I think this line can be reverted for consistency with the rest of `WindowUtils` now that you've made the change elsewhere to pass `setTimeout` directly instead